.github/workflows/standalone-fleet-ci.yml

```
@@ -70,36 +70,37 @@ jobs:
             --agents 3
             --network "nw01"
       -
-        name: K3d import images
+        name: K3d Import Images
         run: |
-          k3d image import rancher/fleet:dev rancher/fleet-agent:dev -m direct
+          k3d image import rancher/fleet:dev rancher/fleet-agent:dev
       -
-        name: Set up tmate debug session
-        if: ${{ github.event_name == 'workflow_dispatch' && github.event.inputs.debug_enabled }}
+        name: Set Up Tmate Debug Session
+        if: ${{ github.event_name == 'workflow_dispatch' && github.event.inputs.enable_tmate }}
         uses: mxschmitt/action-tmate@v3
         timeout-minutes: 15
         with:
           limit-access-to-actor: true
       -
-        name: Deploy standalone fleet
+        name: Deploy Standalone Fleet
```

nit: don't think we need to distinguish standlone fleet, we're either deploying Fleet or deploying Rancher
```suggestion
        name: Deploy Fleet
```